stack.yaml

```diff
@@ -34,8 +34,8 @@ extra-deps:
     - persistent-template
 
 # cardano-crypto
-- git: https://github.com/input-output-hk/cardano-crypto
-  commit: 3c5db489c71a4d70ee43f5f9b979fcde3c797f2a
+- git: https://github.com/Anviking/cardano-crypto
+  commit: a6c3bfe12290cf4f6ac47836a1acb3d417da9d96
```

Comment:
https://github.com/input-output-hk/cardano-crypto/compare/master...Anviking:anviking/encryptedCreateDirectWithoutTweak?expand=0

Reply:
Alternatively, we could also just _forbid_ the use of keys using the derivation scheme #1 since it is known to have security issues.

Reply:
> could also just forbid the use of keys using the derivation scheme #1 since it is known to have security issues

Yes. I'm not sure if we'd want to do that 🤷‍♂. Taking your approval https://github.com/input-output-hk/cardano-wallet/pull/1321#pullrequestreview-364276474 as this way is fine for now though.

Reply:
> Taking your approval #1321 (review) as this way is fine for now though.

Hmmm. No. These are two separate things. I am approving a different PR, not this one :| ... 

So, let's drop "random" from the possible options in the command-line and simply, not expose it for now. Could you comment on the relevant JIRA ticket about this decision please?

Reply:
Done. I dropped the changes to cardano-crypto and extended tests. And commented on jira.

I could have added a "expect failure"-test case for roundtripping keys derived using `DerivationScheme1`, but I deemed it not worth it.